tests/test_soda.py

```diff
@@ -7,7 +7,7 @@
 import json
 
 
-PREFIX = "mock"
+PREFIX = "http://"
```

Comment:
The requests library drops querystring parameters when the protocol is not http/https. ([details](https://bugs.launchpad.net/requests-mock/+bug/1518497))